src/Illuminate/Auth/Middleware/Authenticate.php

```diff
@@ -64,6 +64,20 @@ protected function authenticate($request, array $guards)
             }
         }
 
+        $this->unauthenticated($request, $guards);
+    }
+
+    /**
+     * Handle unauthenticated user.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  array  $gaurds
+     * @return void
+     *
+     * @throws \Illuminate\Auth\AuthenticationException
+     */
+    protected function unauthenticated($request, array $gaurds)
```

Comment:
Typo in method parameter $gaurds

Reply:
```suggestion
    protected function unauthenticated($request, array $guards)
```

Reply:
wow sorry, I don't know how I missed that. thanks!